consensus/hotstuff/event_loop.go

```diff
@@ -1,216 +1,13 @@
 package hotstuff
 
-import (
-	"time"
+import "github.com/onflow/flow-go/model/flow"
 
-	"github.com/rs/zerolog"
+// EventLoop performs buffer and processing of incoming proposals and QCs.
+type EventLoop interface {
```

Comment:
I am wondering why we need this interface. In the following code, you have replaced the `EventLoop` with `module.HotSuff`, which seems right to me ... 
Maybe the difference is that `EventLoop` has the additional method `SubmitTrustedQC` which HotStuff doesn't have? If that is the case, we could write

```golang
type EventLoop interface {
	module.HotStuff
	
	// SubmitTrustedQC accepts QC for processing. QC will be dispatched on worker thread.
	// CAUTION: QC is trusted (_not_ validated again)
	SubmitTrustedQC(qc *flow.QuorumCertificate)
}
```

Reply:
That's one of the reasons. Originally I have added interface to make it uniform with other components. Agree, let's do it this way.